_scripts/clone-l10n.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env sh
+
+set -e
+
+MODULE=$1
+
+if [ -z "$MODULE" ]; then
+  echo "\nYou need specify a module.\n"
+  exit 0
+fi
+
+if [ -n "$FXA_L10N_SKIP" ]; then
+    echo "Skipping fxa-content-server-l10n update..."
+    exit 0
+fi
+
+DOWNLOAD_PATH="https://github.com/mozilla/fxa-content-server-l10n.git"
+SCRIPT_DIR=$(dirname "$0")
+ROOT_DIR=$(realpath "$SCRIPT_DIR/..")
```

Comment:
`realpath` doesn't exist on mac. I think we should be ok using paths relative to `SCRIPT_DIR`